typescript/cli/package.json

```diff
@@ -2,78 +2,75 @@
   "name": "@hyperlane-xyz/cli",
   "version": "12.0.0",
   "description": "A command-line utility for common Hyperlane operations",
-  "dependencies": {
+  "devDependencies": {
     "@aws-sdk/client-kms": "^3.577.0",
     "@aws-sdk/client-s3": "^3.577.0",
+    "@eslint/js": "^9.15.0",
+    "@ethersproject/abi": "*",
+    "@ethersproject/providers": "*",
     "@hyperlane-xyz/registry": "11.1.0",
     "@hyperlane-xyz/sdk": "12.0.0",
     "@hyperlane-xyz/utils": "12.0.0",
     "@inquirer/core": "9.0.10",
     "@inquirer/figures": "1.0.5",
     "@inquirer/prompts": "3.3.2",
     "@inquirer/search": "^3.0.1",
-    "ansi-escapes": "^7.0.0",
-    "asn1.js": "^5.4.1",
-    "bignumber.js": "^9.1.1",
-    "chalk": "^5.3.0",
-    "ethers": "^5.7.2",
-    "latest-version": "^8.0.0",
-    "terminal-link": "^3.0.0",
-    "tsx": "^4.19.1",
-    "yaml": "2.4.5",
-    "yargs": "^17.7.2",
-    "zksync-ethers": "^5.10.0",
-    "zod": "^3.21.2",
-    "zod-validation-error": "^3.3.0",
-    "zx": "^8.1.4"
-  },
-  "devDependencies": {
-    "@eslint/js": "^9.15.0",
-    "@ethersproject/abi": "*",
-    "@ethersproject/providers": "*",
     "@types/chai-as-promised": "^8",
     "@types/mocha": "^10.0.1",
     "@types/node": "^18.14.5",
     "@types/yargs": "^17.0.24",
     "@typescript-eslint/eslint-plugin": "^8.1.6",
     "@typescript-eslint/parser": "^8.1.6",
+    "@vercel/ncc": "^0.38.3",
+    "ansi-escapes": "^7.0.0",
+    "asn1.js": "^5.4.1",
+    "bignumber.js": "^9.1.1",
     "chai": "^4.5.0",
     "chai-as-promised": "^8.0.0",
+    "chalk": "^5.3.0",
     "eslint": "^9.15.0",
     "eslint-config-prettier": "^9.1.0",
     "eslint-import-resolver-typescript": "^3.6.3",
     "eslint-plugin-import": "^2.31.0",
+    "ethers": "^5.7.2",
+    "latest-version": "^8.0.0",
     "mocha": "^10.2.0",
     "prettier": "^2.8.8",
-    "typescript": "5.3.3"
+    "terminal-link": "^3.0.0",
+    "tsx": "^4.19.1",
+    "typescript": "5.3.3",
+    "yaml": "2.4.5",
+    "yargs": "^17.7.2",
+    "zksync-ethers": "^5.10.0",
+    "zod": "^3.21.2",
+    "zod-validation-error": "^3.3.0",
+    "zx": "^8.1.4"
   },
   "scripts": {
     "hyperlane": "node ./dist/cli.js",
     "build": "yarn version:update && tsc",
+    "bundle": "rm -rf ./cli-bundle && ncc build ./dist/cli.js -o cli-bundle && node ./scripts/ncc.post-bundle.mjs",
     "dev": "yarn version:update && tsc --watch",
-    "clean": "rm -rf ./dist",
+    "clean": "rm -rf ./dist && rm -rf ./cli-bundle",
     "lint": "eslint -c ./eslint.config.mjs",
     "prettier": "prettier --write ./src ./examples",
     "test:ci": "yarn mocha --config .mocharc.json",
     "test:e2e": "./scripts/run-e2e-test.sh",
-    "version:update": "echo \"export const VERSION = '$npm_package_version';\" > src/version.ts"
+    "version:update": "echo \"export const VERSION = '$npm_package_version';\" > src/version.ts",
+    "prepack": "yarn bundle"
   },
   "files": [
-    "./dist",
-    "./examples",
-    "package.json"
+    "./cli-bundle",
+    "./examples"
```

Comment:
should we not still include the pjson?

Reply:
Even if not included in the `files` array explicitly, it will be when publishing 
ref https://docs.npmjs.com/cli/v8/configuring-npm/package-json#files:~:text=regardless%20of%20settings%3A-,package.json,-README

Reply:
got it, so this diff is an explicit removal but implicit inclusion? seems better to be explicit

Reply:
makes sense,  i'll add it back!